internal/html/static/css/main.css

```
@@ -35,7 +35,7 @@ nav {
   border-radius: 0.5em;
   display: flex;
 }
-nav a {
+[href="#pkg-index"] {
```

```suggestion
nav a[href="#pkg-index"] {
```
Would this be more specific? 
There's a link to pkg index in the page elsewhere too. (When you hover on the index.)

---

yeah couldn't hurt - thank you sir